examples/acm-terraform-blog-part3/README.md

```diff
@@ -0,0 +1,88 @@
+# Enable ACM features with Terraform - Part 3
+
+This is Part3 of the tutorial to accompany a short series of  blog articles explaining how to enable [Anthos Config Management (ACM)](https://cloud.google.com/anthos/config-management) with Terraform.
+
+In the [first part](../acm-terraform-blog-part1), we explained how to use Teraform to create a cluster and manage its config from git via [Config Sync](https://cloud.google.com/anthos-config-management/docs/config-sync-overview).
+
+In the [second part](../acm-terraform-blog-part2) we added guard rails for the cluster configuration via [Policy Controller](https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller).
+
+In this article we'll demonstrate how, using Config Connector, you can provision your GCP cloud resources following the same GitOps K8s native model.
+
+## Provision GCP resources
+
+1. Set the variable for the project from [part2](../acm-terraform-blog-part2). We will re-use that project but create a new cluster since we cleaned up at the end of the first section. If you are working in a different project, enable required GCP APIs, as described in [part1/README.md](../part1/README.md).
+
+    ```bash
+    PROJECT_ID = [PROJECT_ID]
+    ```
+1. Note that [wordpress-bundle.yaml](./config-root/wordpress-bundle) was updated to use GCP MySQL database. Also we added [configconnector.yaml](./config-root/configconnector.yaml) to initialize the instance of Config Connector add-on on the cluster.
+
+1. Use [kpt](kpt.dev) to customize the `config-root` directory that will be configured as the source of the objects installed on the cluster.
+
+    ```bash
+    kpt fn eval --image gcr.io/kpt-fn/apply-setters:v0.1 ./config-root -- projectId=$PROJECT_ID
+    ```
+1. Submit the updated configuration into your branch.
+1. Ensure that `sync_repo` and `sync_brach` variables are updated in [variables.tf](./terraform/variables.tf)
+1. Before running Terraform, notice the changes in [gke.tf](./terraform/gke.tf):
+     - we are using `beta-public-cluster` module
+     - `config_connector` variable is set to true
+     - we are using `workload-identity` module to create Google Service Account used by Config Connector to create GCP resource and connect it to Kubernetes Service Account that is running in Config Connector `cnrm-system` namespace.
+1. As as in the previous part, create cluster using Terraform:
+
+    ```bash
+    # obtain user access credentials to use for Terraform commands
+    gcloud auth application-default login
+
+    # continue in /terraform directory
+    cd terraform
+
+    terraform init
+    terraform plan -var=project=$PROJECT_ID
```

Comment:
Small note: in general I recommend simply setting the variables either in a tfvars file or by setting an environment variable (`TF_VAR_project`). Setting them only at the command line is very easy to forget / fat finger.